docs/source/user_guide/installation.rst

```diff
@@ -22,13 +22,22 @@ To install from source:
    In most cases, it is recommended to install Raster Loader in a virtual environment.
    Use venv_ to create and manage your virtual environment.
 
-The above will install the dependencies required to work with both BigQuery and Snowflake and. In case you only want to work with one of them, you can install the
+The above will install the dependencies required to work with the different cloud providers (BigQuery, Snowflake, Databricks). In case you only want to work with one of them, you can install the
 dependencies for each of them separately:
 
 .. code-block:: bash
 
    pip install -U raster-loader"[bigquery]"
    pip install -U raster-loader"[snowflake]"
+   pip install -U raster-loader"[databricks]"
+
+For the case of Databricks, additionally you will need to install the databricks-connect_ package corresponding to your Databricks Runtime Version. For example, if your cluster uses DBR 15.1, install:
```

Comment:
Why is this dependency not included in setup.cfg file? It should be installed as a dependency of this extra option (databricks)